security.md

```diff
@@ -69,6 +70,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2017-2672"></a>CVE-2017-2672: audit log stores plain text compute resource image passwords
+
+Compute resource images added/registered in Foreman with username and password credentials will have the password stored in plain text in the audit log. A user with permission to view the audit log can view the password and may gain access to newly provisioned hosts.
+
+This issue was reported by Daniel Kimsey.
+
+* Affects Foreman 1.4 and higher
+* Fix in progress
```

Comment:
Normally this stays as "in progress" until it's released, or assigned a release.